cob_gazebo/CMakeLists.txt

```diff
@@ -5,27 +5,6 @@ find_package(catkin REQUIRED COMPONENTS)
 
 catkin_package()
 
-set(robotlist
-    cob3-3
-    cob3-6
-    cob4-1
-    raw3-1
-    raw3-2
-    raw3-3
-    raw3-4
-    raw3-5
-    raw3-6
-)
-
-### TESTING ###
-if(CATKIN_ENABLE_TESTING)
-    foreach(robot ${robotlist})
-        message("testing for robot: ${robot}")
-        #rosbuild_add_roslaunch_check(launch robot:=${robot})
-        #rosbuild_add_roslaunch_check(launch ROBOT=${robot})
-    endforeach()
-endif()
-
```

Comment:
is is commented because cmake is complaining about arguments. I'd like to keep this and rather investigate why we cannot use roslaunch check at the moment